docs/index.md

```diff
@@ -0,0 +1,58 @@
+[go/evergreen-docs](http://go/evergreen-docs)
```

Comment:
What does this file do? Is it not possible to do something clever like symlink this to `docs/Home.md` to avoid the copy-pasting?

Reply:
Symlinks are a clever solution! done
The challenge here is that pine is currently referencing Home.md ([here](https://github.com/evergreen-ci/pine/blob/340ccaa557929db793abbf6e0e6fd4f7c5189ae8/website/docusaurus.config.js#L196)) and Backstage requires an index.md instead. The plan is to have both for a bit to give a moment to change over pine to use `index.md`. Opened https://github.com/evergreen-ci/pine/pull/219 to skip the new file for now.